Gemfile

```diff
@@ -30,17 +30,17 @@ gem 'ancestry', '2.0.0'
 gem 'gds-api-adapters', '20.1.1'
 gem 'statsd-ruby', '1.1.0'
 gem 'unicorn', '4.9.0'
-gem 'sidekiq', '2.17.2'
-gem 'sidekiq-statsd', '0.1.2'
+gem 'sidekiq', '~> 3.5.0'
+gem 'sidekiq-statsd', '0.1.4'
 gem 'activejob-retry'
 
-gem 'redis', '3.0.6'
+gem 'redis', '~> 3.2.1'
```

Comment:
They don't officially declare support for semver but the changelog seems pretty solid to me unless I'm missing something obvious (as I have been known to do on occasion). They're stating they'll break compatibility with their next major bump to the 4.x series - but that's the point isn't it?